docs/design/Profiles.md

```diff
@@ -0,0 +1,57 @@
+# Profiles
+
+## Overview
+
+Profiles is a configuration format and feature of Zonemaster Engine.
+Each of Zonemaster CLI, Backend and GUI allow users and administrators to configure Zonemaster Engine with different profiles.
+
+This document provides a high-level view of how profiles are applied across the different Zonemaster components.
+For details on how profiles integrate with a given component, refer to the respective component documentations.
+Links to the relevant sections are provided below.
+
+
+## Background
+
+The Zonemaster requirements and specification documents speak of user-configurable profiles and policies for Zonemaster Engine and CLI.
+
+In release 2017.4 all of the configurable features "config", "filter", "policy" and "profile" from the requirements and specifications were integrated into a single data structure, also called "profile".
```

Comment:
An update is needed here. May some placeholder instead of a version?

Reply:
@mattias-p, I second your proposal to have the profile change merged into a feature branch. It makes most sense.

Reply:
Maybe "Previously"? The proper place for documenting changes between releases is the Changes file anyway.

Reply:
I suggest that >>In release ... called "profile".<< is removed and that we make sure there is an issue describing the the change and we link to that from CHANGES.